.pre-commit-config.yaml

```diff
@@ -31,3 +31,13 @@ repos:
           - --format=custom
           - --configfile=tests/bandit.yaml
         files: ^(homeassistant|script|tests)/.+\.py$
+-   repo: https://github.com/pre-commit/mirrors-isort
+    rev: v4.3.21
+    hooks:
+    -   id: isort
+        args:
+          - --multi-line=3
```

Comment:
We should not pass these. It is defined in our `setup.cfg`, not sure at this point if that causes issues for pre-commit having it defined there.

Otherwise, I suggest we create the separate isort configuration file.

In general, let's try to define/configure stuff as centralized as possible, avoiding multiple points of truth.

Reply:
Good point, I've changed it 👍 

I didn't realize that those settings were there.